presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixConfigurationInitializer.java

```diff
@@ -81,7 +80,6 @@ public void updateConfiguration(Configuration config, HdfsContext context, URI u
     {
         if (!cacheReady) {
             setCacheDataEnabled(config, false);
-            setCacheKey(config, "rubix_disabled");
```

Comment:
this needs to stay for now as `cacheReady` is still not guaranteed until first get file system call

Reply:
I do not understand what you mean here.